.eslintrc.js

```diff
@@ -80,6 +80,14 @@ module.exports = {
     'simple-import-sort/imports': 'error',
     'simple-import-sort/exports': 'error',
   },
+  overrides: [
+    {
+      files: 'bin/**',
```

Comment:
Can we add a small comment here explaining why we're using this?
```suggestion
      // the stencil entry point still uses `var`, ignore errors related to it
      files: 'bin/**',
```